src/main/java/br/com/moip/Client.java

```diff
@@ -172,4 +180,52 @@ private StringBuilder readBody(final InputStream inputStream) throws IOException
 
         return body;
     }
+
+    private String getBody(Object object, ContentType contentType) {
+        if (contentType == ContentType.APPLICATION_FORM_URLENCODED) {
+            return jsonToUrlEncodedString((JsonObject) new JsonParser().parse(gson.toJson(object)));
+        }
+
+        return gson.toJson(object);
+    }
+
+    public Authentication getAuthentication() {
+        return authentication;
+    }
+
+    public static String jsonToUrlEncodedString(JsonObject jsonObject) {
+        return jsonToUrlEncodedString(jsonObject, "");
+    }
+
+    private static String jsonToUrlEncodedString(JsonObject jsonObject, String prefix) {
```

Comment:
Não tem alguma lib que faça isso pra vc? Apache alguma coisa? Se tiver, seria mais legal adicioanr a dependencia to que ter esse código aqui :)

Reply:
A ideia inicial era achar uma lib que converteria um POJO pro url encoded, não encontrei, apenas algo parecido usando Spring. Depois procurei pra converter JSON em url encoded e não encontrei também, por isso adicionei esses métodos.